grammars/asciidoc.cson

```diff
@@ -291,21 +300,47 @@ patterns: [
     'captures':
       '1': 'name': 'markup.list.bullet.asciidoc'
   }
+
+  # Includes inline patterns, see repository section below.
+  {
+    'include': '#inline'
```

Comment:
Good for you. I have to get used to 'generator mode'. Glad it isn't as nested and badly named as Ansible deploy definitions.
I've moved the include as reordering was the most neat and convenient way to get around the issue of bulleted lists getting recognized as bold content. Negatively matching the bullets of the list results in a very nasty regex, which is hard to debug. The required complexity was over my head at least. So by reordering, the lists are matched first, and second bold content is located. This works beautifully, and I haven't discovered any breakage as a result of the reordering.